content/en/digital_certificate.md

```diff
@@ -7,14 +7,14 @@ category: Concept
 ## What it is
 Digital certificates or public key certificates are issued by a certificate authority (CA). These certificates are electronic files or passwords that allow entities to exchange data or resources over the internet securely.
 
-Certificates authenticate the user through encryption methods and a set of roles and policies. Just as a passport certifies an individual's identity, a digital certificate establishes the user's identity. Note that a user can be a person or service within a [microservice](https://glossary.cncf.io/microservices/) app. 
+Digital certificate provides confirmation that a website is genuine and that documents and emails are authentic. This projects public trust, assuring clients that they are dealing with a genuine company that values their security and privacy.. Just as a passport certifies an individual's identity, a digital certificate establishes the user's identity. Note that a user can be a person or service within a [microservice](https://glossary.cncf.io/microservices/) app. 
```

Comment:
I don't think this example works. Certificates can't tell me if the website I'm on is what it claims it is — the internet is full of fraudulent offerings. That aside, Jason is asking to go into the identity and encryption mechanisms for TLS and make it broader so we aren't only focusing on user authentication. 

If this is not something you are familiar with, no worries. Please say so and we'll get someone else to help with that.

Reply:
@CathPag , I am sorry I have lot of struggle to fix what's asked from me. Yes please, I wish for someone to help me with that

Reply:
No worries. We'll take it from here :)